*** tests/A-13-13B-M-ok.out	Tue Nov  1 21:35:57 2022
--- tests/A-13-13B-M-ok.outhyp	Tue Nov 15 17:32:41 2022
***************
*** 58,64 ****
  10 - Terminar comunicação activa
  11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: Duração da comunicação (em minutos): Custo da comunicação: 400 unidades.
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
--- 58,64 ----
  10 - Terminar comunicação activa
  11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: Duração da comunicação (em minutos): Custo da comunicação: 800 unidades.
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
***************
*** 92,98 ****
  5 - Desactivar notificações
  6 - Mostrar saldo de cliente
  0 - Sair
! Escolha uma opção: CLIENT|cli001|Manuel Pinheiro|103443|GOLD|YES|4|520|400
  CLIENT|cli002|Pedro Pinheiro|103447|NORMAL|YES|2|0|0
  CLIENT|cli003|Pedro Oliveira|103449|NORMAL|YES|3|0|0
  CLIENT|cli201|Ludgero Oliveira|103440|NORMAL|YES|0|0|0
--- 92,98 ----
  5 - Desactivar notificações
  6 - Mostrar saldo de cliente
  0 - Sair
! Escolha uma opção: CLIENT|cli001|Manuel Pinheiro|103443|NORMAL|YES|4|0|1320
  CLIENT|cli002|Pedro Pinheiro|103447|NORMAL|YES|2|0|0
  CLIENT|cli003|Pedro Oliveira|103449|NORMAL|YES|3|0|0
  CLIENT|cli201|Ludgero Oliveira|103440|NORMAL|YES|0|0|0
***************
*** 140,146 ****
  10 - Terminar comunicação activa
  11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: Duração da comunicação (em minutos): Custo da comunicação: 140 unidades.
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
--- 140,146 ----
  10 - Terminar comunicação activa
  11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: Duração da comunicação (em minutos): Custo da comunicação: 210 unidades.
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
***************
*** 174,180 ****
  5 - Desactivar notificações
  6 - Mostrar saldo de cliente
  0 - Sair
! Escolha uma opção: CLIENT|cli001|Manuel Pinheiro|103443|NORMAL|YES|4|520|540
  CLIENT|cli002|Pedro Pinheiro|103447|NORMAL|YES|2|0|0
  CLIENT|cli003|Pedro Oliveira|103449|NORMAL|YES|3|0|0
  CLIENT|cli201|Ludgero Oliveira|103440|NORMAL|YES|0|0|0
--- 174,180 ----
  5 - Desactivar notificações
  6 - Mostrar saldo de cliente
  0 - Sair
! Escolha uma opção: CLIENT|cli001|Manuel Pinheiro|103443|NORMAL|YES|4|0|1530
  CLIENT|cli002|Pedro Pinheiro|103447|NORMAL|YES|2|0|0
  CLIENT|cli003|Pedro Oliveira|103449|NORMAL|YES|3|0|0
  CLIENT|cli201|Ludgero Oliveira|103440|NORMAL|YES|0|0|0
