*** tests/A-13-18-M-ok.out	Thu Nov  3 21:26:18 2022
--- tests/A-13-18-M-ok.outhyp	Tue Nov 15 17:32:43 2022
***************
*** 42,48 ****
  10 - Terminar comunicação activa
  11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: Duração da comunicação (em minutos): Custo da comunicação: 20 unidades.
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
--- 42,48 ----
  10 - Terminar comunicação activa
  11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: Duração da comunicação (em minutos): Custo da comunicação: 30 unidades.
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
***************
*** 66,72 ****
  10 - Terminar comunicação activa
  11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: Duração da comunicação (em minutos): Custo da comunicação: 20 unidades.
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
--- 66,72 ----
  10 - Terminar comunicação activa
  11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: Duração da comunicação (em minutos): Custo da comunicação: 30 unidades.
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
***************
*** 90,96 ****
  10 - Terminar comunicação activa
  11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: Duração da comunicação (em minutos): Custo da comunicação: 20 unidades.
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
--- 90,96 ----
  10 - Terminar comunicação activa
  11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: Duração da comunicação (em minutos): Custo da comunicação: 30 unidades.
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
***************
*** 114,120 ****
  10 - Terminar comunicação activa
  11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: Duração da comunicação (em minutos): Custo da comunicação: 20 unidades.
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
--- 114,120 ----
  10 - Terminar comunicação activa
  11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: Duração da comunicação (em minutos): Custo da comunicação: 30 unidades.
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
***************
*** 127,133 ****
  9 - Iniciar comunicação
  11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: Valores para o terminal '969011':0 (pagamentos),  80 (dívidas).
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
--- 127,133 ----
  9 - Iniciar comunicação
  11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: Valores para o terminal '969011':0 (pagamentos),  120 (dívidas).
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
***************
*** 161,167 ****
  5 - Desactivar notificações
  6 - Mostrar saldo de cliente
  0 - Sair
! Escolha uma opção: CLIENT|cli001|Manuel Pinheiro|103443|GOLD|YES|4|520|80
  CLIENT|cli002|Pedro Pinheiro|103447|NORMAL|YES|2|0|0
  CLIENT|cli003|Pedro Oliveira|103449|NORMAL|YES|3|0|0
  CLIENT|cli201|Ludgero Oliveira|103440|NORMAL|YES|0|0|0
--- 161,167 ----
  5 - Desactivar notificações
  6 - Mostrar saldo de cliente
  0 - Sair
! Escolha uma opção: CLIENT|cli001|Manuel Pinheiro|103443|NORMAL|YES|4|0|640
  CLIENT|cli002|Pedro Pinheiro|103447|NORMAL|YES|2|0|0
  CLIENT|cli003|Pedro Oliveira|103449|NORMAL|YES|3|0|0
  CLIENT|cli201|Ludgero Oliveira|103440|NORMAL|YES|0|0|0
***************
*** 209,215 ****
  10 - Terminar comunicação activa
  11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: Duração da comunicação (em minutos): Custo da comunicação: 460 unidades.
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
--- 209,215 ----
  10 - Terminar comunicação activa
  11 - Mostrar comunicação em curso
  0 - Sair
! Escolha uma opção: Duração da comunicação (em minutos): Custo da comunicação: 690 unidades.
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
***************
*** 243,249 ****
  5 - Desactivar notificações
  6 - Mostrar saldo de cliente
  0 - Sair
! Escolha uma opção: CLIENT|cli001|Manuel Pinheiro|103443|NORMAL|YES|4|520|540
  CLIENT|cli002|Pedro Pinheiro|103447|NORMAL|YES|2|0|0
  CLIENT|cli003|Pedro Oliveira|103449|NORMAL|YES|3|0|0
  CLIENT|cli201|Ludgero Oliveira|103440|NORMAL|YES|0|0|0
--- 243,249 ----
  5 - Desactivar notificações
  6 - Mostrar saldo de cliente
  0 - Sair
! Escolha uma opção: CLIENT|cli001|Manuel Pinheiro|103443|NORMAL|YES|4|0|1330
  CLIENT|cli002|Pedro Pinheiro|103447|NORMAL|YES|2|0|0
  CLIENT|cli003|Pedro Oliveira|103449|NORMAL|YES|3|0|0
  CLIENT|cli201|Ludgero Oliveira|103440|NORMAL|YES|0|0|0
