*** tests/A-22-01-M-ok.out	Mon Oct 31 08:40:15 2022
--- tests/A-22-01-M-ok.outhyp	Tue Nov 15 17:33:10 2022
***************
*** 57,64 ****
  6 - Mostrar terminais sem actividade
  7 - Mostrar terminais com saldo positivo
  0 - Sair
! Escolha uma opção: Identificador do cliente: TEXT|1|969015|969014|5|10|FINISHED
! CONSULTAS
  1 - Mostrar todas as comunicações
  2 - Mostrar comunicações feitas por um cliente
  3 - Mostrar comunicações recebidas por um cliente
--- 57,63 ----
  6 - Mostrar terminais sem actividade
  7 - Mostrar terminais com saldo positivo
  0 - Sair
! Escolha uma opção: Identificador do cliente: CONSULTAS
  1 - Mostrar todas as comunicações
  2 - Mostrar comunicações feitas por um cliente
  3 - Mostrar comunicações recebidas por um cliente
