*** tests/A-23-02-M-ok.out	Mon Oct 31 19:32:00 2022
--- tests/A-23-02-M-ok.outhyp	Tue Nov 15 17:33:14 2022
***************
*** 121,127 ****
  6 - Mostrar terminais sem actividade
  7 - Mostrar terminais com saldo positivo
  0 - Sair
! Escolha uma opção: CLIENT|cli003|Pedro Oliveira|103449|NORMAL|YES|1|40|20
  CONSULTAS
  1 - Mostrar todas as comunicações
  2 - Mostrar comunicações feitas por um cliente
--- 121,127 ----
  6 - Mostrar terminais sem actividade
  7 - Mostrar terminais com saldo positivo
  0 - Sair
! Escolha uma opção: CLIENT|cli003|Pedro Oliveira|103449|NORMAL|YES|1|0|60
  CONSULTAS
  1 - Mostrar todas as comunicações
  2 - Mostrar comunicações feitas por um cliente
