*** testsBoth/A-13-06-M-ok.out	Sun Nov  6 22:59:07 2022
--- testsBoth/A-13-06-M-ok.outhyp	Tue Nov 15 17:19:20 2022
***************
*** 95,101 ****
  0 - Sair
  Escolha uma opção: 1
  Identificador do cliente: cli001
! CLIENT|cli001|Manuel Pinheiro|103443|GOLD|YES|4|600|0
  GESTÃO DE CLIENTES
  1 - Visualizar cliente
  2 - Visualizar todos os clientes
--- 95,101 ----
  0 - Sair
  Escolha uma opção: 1
  Identificador do cliente: cli001
! CLIENT|cli001|Manuel Pinheiro|103443|NORMAL|YES|4|0|600
  GESTÃO DE CLIENTES
  1 - Visualizar cliente
  2 - Visualizar todos os clientes
***************
*** 105,111 ****
  6 - Mostrar saldo de cliente
  0 - Sair
  Escolha uma opção: 2
! CLIENT|cli001|Manuel Pinheiro|103443|GOLD|YES|4|600|0
  CLIENT|cli002|Pedro Pinheiro|103447|NORMAL|YES|2|0|0
  CLIENT|cli003|Pedro Oliveira|103449|NORMAL|YES|3|0|0
  CLIENT|cli201|Ludgero Oliveira|103440|NORMAL|YES|0|0|0
--- 105,111 ----
  6 - Mostrar saldo de cliente
  0 - Sair
  Escolha uma opção: 2
! CLIENT|cli001|Manuel Pinheiro|103443|NORMAL|YES|4|0|600
  CLIENT|cli002|Pedro Pinheiro|103447|NORMAL|YES|2|0|0
  CLIENT|cli003|Pedro Oliveira|103449|NORMAL|YES|3|0|0
  CLIENT|cli201|Ludgero Oliveira|103440|NORMAL|YES|0|0|0
