*** testsBoth/A-13-13B-M-ok.out	Sun Nov  6 22:59:08 2022
--- testsBoth/A-13-13B-M-ok.outhyp	Tue Nov 15 17:19:22 2022
***************
*** 70,76 ****
  0 - Sair
  Escolha uma opção: 10
  Duração da comunicação (em minutos): 40
! Custo da comunicação: 400 unidades.
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
--- 70,76 ----
  0 - Sair
  Escolha uma opção: 10
  Duração da comunicação (em minutos): 40
! Custo da comunicação: 800 unidades.
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
***************
*** 108,114 ****
  6 - Mostrar saldo de cliente
  0 - Sair
  Escolha uma opção: 2
! CLIENT|cli001|Manuel Pinheiro|103443|GOLD|YES|4|520|400
  CLIENT|cli002|Pedro Pinheiro|103447|NORMAL|YES|2|0|0
  CLIENT|cli003|Pedro Oliveira|103449|NORMAL|YES|3|0|0
  CLIENT|cli201|Ludgero Oliveira|103440|NORMAL|YES|0|0|0
--- 108,114 ----
  6 - Mostrar saldo de cliente
  0 - Sair
  Escolha uma opção: 2
! CLIENT|cli001|Manuel Pinheiro|103443|NORMAL|YES|4|0|1320
  CLIENT|cli002|Pedro Pinheiro|103447|NORMAL|YES|2|0|0
  CLIENT|cli003|Pedro Oliveira|103449|NORMAL|YES|3|0|0
  CLIENT|cli201|Ludgero Oliveira|103440|NORMAL|YES|0|0|0
***************
*** 165,171 ****
  0 - Sair
  Escolha uma opção: 10
  Duração da comunicação (em minutos): 7
! Custo da comunicação: 140 unidades.
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
--- 165,171 ----
  0 - Sair
  Escolha uma opção: 10
  Duração da comunicação (em minutos): 7
! Custo da comunicação: 210 unidades.
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
***************
*** 203,209 ****
  6 - Mostrar saldo de cliente
  0 - Sair
  Escolha uma opção: 2
! CLIENT|cli001|Manuel Pinheiro|103443|NORMAL|YES|4|520|540
  CLIENT|cli002|Pedro Pinheiro|103447|NORMAL|YES|2|0|0
  CLIENT|cli003|Pedro Oliveira|103449|NORMAL|YES|3|0|0
  CLIENT|cli201|Ludgero Oliveira|103440|NORMAL|YES|0|0|0
--- 203,209 ----
  6 - Mostrar saldo de cliente
  0 - Sair
  Escolha uma opção: 2
! CLIENT|cli001|Manuel Pinheiro|103443|NORMAL|YES|4|0|1530
  CLIENT|cli002|Pedro Pinheiro|103447|NORMAL|YES|2|0|0
  CLIENT|cli003|Pedro Oliveira|103449|NORMAL|YES|3|0|0
  CLIENT|cli201|Ludgero Oliveira|103440|NORMAL|YES|0|0|0
