*** testsBoth/A-20-05-M-ok.out	Sun Nov  6 22:59:16 2022
--- testsBoth/A-20-05-M-ok.outhyp	Tue Nov 15 17:19:41 2022
***************
*** 114,119 ****
--- 114,120 ----
  0 - Sair
  Escolha uma opção: 6
  Número da comunicação: 1
+ O identificador da comunicação é inválido.
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
***************
*** 127,133 ****
  11 - Mostrar comunicação em curso
  0 - Sair
  Escolha uma opção: 7
! Valores para o terminal '969011':130 (pagamentos),  0 (dívidas).
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
--- 128,134 ----
  11 - Mostrar comunicação em curso
  0 - Sair
  Escolha uma opção: 7
! Valores para o terminal '969011':90 (pagamentos),  40 (dívidas).
  OPERAÇÕES DO TERMINAL
  1 - Ligar
  2 - Colocar no silêncio
